addOns/authhelper/src/main/javahelp/org/zaproxy/addon/authhelper/resources/help/contents/auth-req-id.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Authentication Request Identification
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Authentication Request Identification</H1>
+
+This <a href="authhelper.html">add-on</a> includes a passive scan rule which attempts to identify authentication requests.<br>
+It identifies authentication requests by the presence of commonly used username and password field names.
```

Comment:
Not that our help content gets much traction/attention but shall we add a request here to people to send ideas/details when they notice something missed?